minio/minio
# Fix site removal API error handling

when site being removed is missing replication config. This can
happen when a new deployment is brought in place of a site that
is lost/destroyed and need to delink old deployment from site
replication.

## Description


## Motivation and Context
Ability to recover from DR scenario

## How to test this PR?
1. set up site replication on clusters sitea, siteb, sitec with `mc admin replicate add` 
2. remove all backend disks of sitea after some operations and start a fresh minio cluster 
3. `mc admin replicate remove siteb sitea --force` should remove the old deployment id for sitea from the site replication config
 

## Types of changes

## Checklist:
